packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -765,8 +794,8 @@ export class Project extends ProjectBase {
   }
 
   private renderEnvironment(env: BuildEnvironment = {},
-                            projectVars: { [name: string]: BuildEnvironmentVariable } = {}):
-      CfnProject.EnvironmentProperty {
+    projectVars: { [name: string]: BuildEnvironmentVariable } = {}):
```

Comment:
I believe this will not pass our linter, which checks for aligned arguments when they span multiple lines.